.github/workflows/tests.yml

```diff
@@ -24,9 +24,13 @@ jobs:
         xcode_version: [15.2]
         virtual_frameworks: [true, false]
         sandbox: [true, false]
+        latest_rules: [true, false]
```

Comment:
I want to run it so that any future PRs can ensure thats green if they intended to fix the issue. I just wont mark it required in github so it wont block merges, thoughts?

Reply:
Yeah, that sounds reasonable. Looks like the latest rules don't work with bazel 6.5